### PR TITLE
More Charm adjustments

### PR DESCRIPTION
**Pull request by ymkc**
- made only normal mobs charmable, excluding battlefield/dynamis/nm/fished up/etc
- added mobmod_charmable to wings of fury furies.lua

### CODE REVIEW

**Comment by xdemolish** [comment]
looks good but i'll let @teschnei merge in case he wants anything done differently

**Comment by ymkc** [comment]
rebased on master

anything wrong/could be better with this?

File: src/map/utils/zoneutils.cpp [hunk 1]
```diff
@@ -500,7 +500,9 @@ void LoadMOBList()
                 PMob->m_TrueDetection = Sql_GetUIntData(SqlHandle, 65);
                 PMob->m_Detects = Sql_GetUIntData(SqlHandle, 66);
 
-                PMob->setMobMod(MOBMOD_CHARMABLE, Sql_GetUIntData(SqlHandle, 67));
+                if (GetZone(ZoneID)->GetType() != ZONETYPE_DYNAMIS && GetZone(ZoneID)->GetType() != ZONETYPE_BATTLEFIELD &&
```
Comment (bendangelo) [inline src/map/utils/zoneutils.cpp]: I think these checks should be done inside `mobutils::calculatestats`. There you can change the charmable value just before the mob spawns. It's just easier to debug when all these checks are done in one place.

**Comment by ymkc** [comment]
ok so leave setMobMod in zoneutils/instance_loader as it is (set it for all mobs) and remove/change it later in mobutils::calculatestats on special mobs( dyna, battlefield, etc) in mobutils::calculatestats?

**Comment by bendangelo** [comment]
You should move all those checks to `mobutils`. You can already find some examples in it. Once you do that I think it's good. 👍

**Comment by ymkc** [comment]
moved it all to modutils, looks much cleaner

**Pull request merged**